docgen/src/examples/tourism/App.js

```diff
@@ -234,11 +239,15 @@ const RoomType = RefinementList.connect(({items, refine, selectedItems}) => {
 });
 
 function Price() {
+  const theme = _.entries(Range.defaultTheme).reduce((acc, i) => {
```

Comment:
Yep I wonder if we should provide it or not as an utility. For now maybe just explain how to do it in the theming guide and let's revisit it after.